### PR TITLE
Adding Information Prerequisites Section

### PR DESCRIPTION
**Pull request by Radhika-Saini**
PR Checklist:

- [ ] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [ ] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [ ] put any other relevant information below

### CODE REVIEW

**Comment by Radhika-Saini** [comment]
want to do some more changes

**Comment by Radhika-Saini** [comment]
I am Radhika Saini, an outreachy Applicant. 
Work on my 1st issue [#1346](https://github.com/conda-forge/conda-forge.github.io/issues/1346) and Requested PR.

File: src/README.md [hunk 1]
```diff
@@ -1,15 +1,120 @@
...
 Documentation update PRs should only change the files in ``src``. 
 
 Once the PR is merged to ``master``, you can find the auto-generated html files in ``docs`` directory.
 
+
+### Code of conduct
+
+We adheres to the [NumFOCUS Code of Conduct](https://numfocus.org/code-of-conduct).
+
+1. Be kind to others. Do not insult or put down others. Behave professionally.
```
[the remark this answers is not in the document]
Comment (kathatherine) [reply src/README.md:37]: Ah, I missed coming back to this. Apologies. Turns out this is all fine. NumFocus has a [template](https://github.com/numfocus/numfocus/blob/8759e21481552f213489e3718979ccecf68e9ead/manual/numfocus-coc.md) you can use for codes of conduct, I've learned. I'll finish the review now.

File: src/README.md [hunk 2]
```diff
@@ -1,15 +1,120 @@
...
+
+1. Be kind to others. Do not insult or put down others. Behave professionally. 
+2. Remember that harassment and sexist, racist, or exclusionary jokes are not appropriate for conda-forge.
+3. All communication should be appropriate for a professional audience including people of many different backgrounds. Sexual language and imagery is not appropriate.
+4. conda-forge is dedicated to providing a harassment-free community for everyone, regardless of gender, sexual orientation, gender identity and expression, disability, physical appearance, body size, race, or religion. We do not tolerate harassment of community members in any form.
+5. We strive to Be empathetic, welcoming, friendly, and patient, Be collaborative, Be inquisitive, Be careful in the words that we choose.
+
+Thank you for helping make this a welcoming, friendly community for all.
+
+This is the short version of Code of Conduct. If you want to read more about it check out on [NumFOCUS Code of Conduct](https://numfocus.org/code-of-conduct).
```
Comment (kathatherine) [inline src/README.md:45]: Comma needed between "it" and "check".

Also replace "on" after "check out" with "the".

"If you want to read more about it, check out the NumFOCUS Code of Conduct."

File: src/README.md [hunk 3]
```diff
@@ -1,15 +1,120 @@
...
+
+
+### Reporting Guidelines
+
+If you believe someone is violating the code of conduct, please report this in a timely manner. Code of conduct violations reduce the value of the community for everyone. The team at conda-forge takes reports of misconduct very seriously and is committed to preserving and maintaining the welcoming nature of our community.
+
+All reports will be kept confidential. Please go through it thoroughly [Reporting Guidelines](https://numfocus.org/code-of-conduct#reporting-guidelines).
+
+
```
Comment (kathatherine) [inline src/README.md:54]: There's a lot of unnecessary blank spaces here.

File: src/README.md [hunk 4]
```diff
@@ -1,15 +1,120 @@
...
+We will attempt to ensure your safety and help with any immediate needs, particularly at an in-person event. We will make every effort to acknowledge receipt within 24 hours (and we’ll aim for much more quickly than that). Have a look at the process of [What Happens After a Report is Filed?](https://numfocus.org/code-of-conduct#enforcement).
+
+ 
+## Reviewing the Report
+We will review the incident as quickly as possible and determine:
+
+1. Whether this is an ongoing situation, or if there is a threat to anyone’s physical safety
+2. What happened
+3. Whether this event constitutes a code of conduct violation
+4. Who the bad actor was, if any
```
Comment (kathatherine) [inline src/README.md:71]: Put a line between item 4 and the sentence on line 72.

File: src/README.md [hunk 5]
```diff
@@ -1,15 +1,120 @@
...
+3. Warning the person to cease their behavior and that any further reports will result in sanctions.
+4. A public announcement that an incident occurred.
+5. Mediation (only if both reporter and reportee agree).
+6. An imposed vacation (e.g. asking someone to “take a week off” from a mailing list).
+7. A permanent or temporary ban from some or all NumFOCUS spaces (mailing lists, GitHub repos, in-person events, etc.).
+8. Assistance to the complainant with a report to other bodies, for example, institutional offices or appropriate law enforcement agencies.
+9. Removing a person from our membership or other formal affiliation.
+10. Publishing an account of the harassment and calling for the resignation of the alleged harasser from their responsibilities (usually pursued by people without formal authority: may be called for if the person is the event leader, or refuses to stand aside from the conflict of interest, or similar).
+11. Any other response that we deem necessary and appropriate to the situation.
+At NumFOCUS events, if a participant engages in behavior that violates this code of conduct, the conference organizers and staff may take any action they deem appropriate..
```
Comment (kathatherine) [inline src/README.md:94]: Same here. A blank line needed between item 11 and the sentence below it.

File: src/README.md [hunk 6]
```diff
@@ -1,15 +1,120 @@
...
+3. Ending a talk that violates the policy early
+4. Not publishing the video or slides of a talk that violated the policy
+5. Not allowing a speaker who violated the policy to give (further) talks at the event now or in the future
+6. Immediately ending any event volunteer responsibilities and privileges the reported person holds
+7. Requiring that the person not volunteer for future events  (either indefinitely or for a certain time period)
+8. Expelling the person from the event without a refund
+9. Requiring that the person immediately leave the event and not return
+10. Banning the person from future events (either indefinitely or for a certain time period)
+11. Any other response that we deem necessary and appropriate to the situation
+No one espousing views or values contrary to the standards of our code of conduct will be permitted to hold any position representing us, including volunteer positions. We have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned with this code of conduct.
```
Comment (kathatherine) [inline src/README.md:109]: And here as well. Blank line between item 11 and the paragraph below it.

File: src/README.md [hunk 7]
```diff
@@ -1,15 +1,120 @@
...
+
+We will contact the person who is the subject of the report to let them know what actions will be taken as a result of the report, if any.
+
+_This code-of-conduct is common to the any projects of conda-forge found here(https://github.com/conda-forge)_
+
+Our policy is to make sure that everyone aware of the initial incident is also made aware that official action has been taken, while still respecting the privacy of individuals. We may choose to make a public report of the incident, while maintaining the anonymity of those involved.
+
```
Comment (kathatherine) [inline src/README.md:116]: Blank lines at the end of this file should be deleted.

**Review by kathatherine — CHANGES_REQUESTED** [review]
Okay, review complete! Let me know if you have any questions.

**Comment by Radhika-Saini** [comment]
Okays, Thanks for the review. Good to see Template, it helps and I will do the changes.

**Pull request closed without merging**